spec/timezone.html

```diff
@@ -214,7 +214,7 @@ <h1>Temporal.TimeZone.prototype.getAbsoluteFor ( _dateTime_ [ , _options_ ] )</h
             1. Return _absolute_.
           1. If _disambiguation_ is *"reject"*, then
             1. Throw a *RangeError* exception.
-        1. <mark>TODO.</mark>
+        1. <mark>TODO - handle skipped times like Spring DST where "compatible" means "later"</mark>
```

Comment:
```suggestion
        1. <mark>TODO - handle n = 0</mark>
```
(I found the comment confusing since there is an entire DST transition not handled, not just "compatible" → "later" :smile: )

Reply:
OK, I updated to `handle n=0 (skipped times like Spring DST)` to remove the confusing part but leave a bread crumb hint about why n=0 happens.